src/main/scala/org/apache/spark/sql/delta/storage/FileSystemLogStore.scala

```diff
@@ -52,7 +52,7 @@ abstract class FileSystemLogStore(
     }
   }
 
-  override def listFrom(path: Path): Iterator[FileStatus] = {
+  override def iteratorFrom(path: Path): Iterator[FileStatus] = {
```

Comment:
Can you please revert this change? The `listFrom` here doesn't correspond to returning a `List`. It's corresponding to a file listing action on a storage system. There's no reason to break a developer API here

Reply:
@brkyvz Oh, I see. thanks. I'll revert.